Source/Pages/Gallery/LibraryMediaManager.swift

```diff
@@ -234,6 +237,15 @@ open class LibraryMediaManager {
                 } else {
                     // transfer the transform so the video renders in the correct orientation
                     videoCompositionTrack.preferredTransform = transform
+                    videoComposition?.renderSize = videoSize
```

Comment:
On line 236, can probably clean up that comment right? I think we determined that `videoComposition?.renderSize = cropRect.size` is needed? (based on Slack convo)

Reply:
Oh yeah sure, I'll clean it up.

Reply:
Updated now.